app/src/main/java/org/fossasia/phimpme/data/local/TrashBinRealmModel.java

```diff
@@ -3,16 +3,35 @@
 import io.realm.RealmObject;
 import io.realm.annotations.PrimaryKey;
 
-/** Created by saurav on 21/6/18. */
+/** Created by saurav on 21/6/18. Last Modified by SatyaJiit */
```

Comment:
I don't think there's a need to add `Last modified by ....`. You'll be modifying several files across all prs you make. There's no need to mention this, particularly unless you create a new class.

Reply:
Really sorry for adding this line. @yashk2000

Reply:
Removed the Comment.